hooks/tk-hiero-export/hiero_translate_template.py

```diff
@@ -39,47 +38,36 @@ def execute(self, template, output_type, **kwargs):
         """
         # First add in any relevant fields from the context
         fields = self.parent.context.as_template_fields(template)
-        mapping = dict([("{%s}" % k, v) for k, v in fields.iteritems()])
-        
-        # Next update the mapping with variables from the session
-        mapping.update({
-            "{Sequence}": "{sequence}",
-            "{Shot}": "{shot}",
-            "{name}": "{project}",
-            "{output}": "{clip}",
-            "{version}": "{tk_version}",
-            "{extension}": "{ext}"
-        })
+        # Substitute the variables that Hiero will inject during export
+        hiero_fields = {
+            "Sequence": "{sequence}",
+            "Shot": "{shot}",
+            "name": "{project}",
+            "output": "{clip}",
+            "width": "{width}",
+            "height": "{height}",
+            "version": "{tk_version}",
+            "extension": "{ext}"
+        }
+        fields.update(hiero_fields)
 
-        # get the string representation of the template object
-        template_str = template.definition
+        for name, key in template.keys.iteritems():
+            if isinstance(key, sgtk.templatekey.SequenceKey):
+                fields.update({name: "FORMAT:#"})
```

Comment:
I know this is my own coding I'm fixing, but I realized this morning this should just be `fields[name] = "FORMAT:#"`